'CHPL_LAUNCHER=pbs-aprun' does not build due to compiler warning of 'snprintf'

Simple fix.

```
launch-pbs-aprun.c: In function 'chpl_launch_cleanup':
launch-pbs-aprun.c:396:59: error: '%s' directive output may be truncated writing up to 4095 bytes into a region of size 993 [-Werror=format-truncation=]
       snprintf(msg, 1024, "Error removing temporary file '%s': %s",
                                                           ^~
                expectFilename, strerror(errno));
                ~~~~~~~~~~~~~~                              
launch-pbs-aprun.c:396:7: note: 'snprintf' output 35 or more bytes (assuming 4130) into a destination of size 1024
       snprintf(msg, 1024, "Error removing temporary file '%s': %s",
       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                expectFilename, strerror(errno));
                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Function with Error**

https://github.com/chapel-lang/chapel/blob/fa3c2c2c691f7e83439509ce0c3ef8d089321f6a/runtime/src/launch/pbs-aprun/launch-pbs-aprun.c#L392-L401

**Declaration of `expectedFilename`**

https://github.com/chapel-lang/chapel/blob/fa3c2c2c691f7e83439509ce0c3ef8d089321f6a/runtime/src/launch/pbs-aprun/launch-pbs-aprun.c#L52

**Buffer read into**

https://github.com/chapel-lang/chapel/blob/fa3c2c2c691f7e83439509ce0c3ef8d089321f6a/runtime/src/launch/pbs-aprun/launch-pbs-aprun.c#L395

Replacing this with...

```c
char msg[FILENAME_MAX];
```

May solve the problem.

--- a/runtime/src/launch/pbs-aprun/launch-pbs-aprun.c
+++ b/runtime/src/launch/pbs-aprun/launch-pbs-aprun.c
@@ -392,14 +392,13 @@ static void genQsubScript(int argc, char *argv[], int numLocales) {
 static void chpl_launch_cleanup(void) {
   if (!debug) {
     if (unlink(expectFilename)) {
-      char msg[1024];
-      snprintf(msg, 1024, "Error removing temporary file '%s': %s",
+      char msg[FILENAME_MAX + 35];
+      snprintf(msg, FILENAME_MAX + 35, "Error removing temporary file '%s': %s",
                expectFilename, strerror(errno));
       chpl_warning(msg, 0, 0);
     }
   }
 }
-
 
 int chpl_launch(int argc, char* argv[], int32_t numLocales) {
   int retcode;

